### DISCUSSION TITLE
Make the Server New Player Friendly

### DISCUSSION THREAD

**Discussion by Scrublettt** [chromiecraft/chromiecraft#5181] [Suggestions]
Isnt getting maximum players on the current bracket in order to test and report bugs the goal? If so, why dont you think about giving an additional boost through classic levels so its new player friendly? Currently the gap is getting bigger and bigger and with TBC requiring quite a lot of farm (reps and stuff) it may not be appealing for a new player to join and catch up.

**Comment by balleny** [comment]
It is not.
Having new players work through vanilla content ensures that the server has also regression checks taking place.
